homeworks/hw2/Trie/Program.cs

```diff
@@ -0,0 +1,75 @@
+﻿using TrieHomework;
+
+static void PrintActions()
+{
+    Console.WriteLine("0 - exit\n1 - add\n2 - check if contains\n3 - remove\n" +
+            "4 - count how many words with this prefix\n" +
+            "5 - find out the number of words in trie\n6 - print actions again");
+}
+
+if (!Trie.Tests())
+{
+    Console.WriteLine("Tests failed");
+    return;
+}
+Trie trie = new Trie();
```

Comment:
```suggestion
var trie = new Trie();
```
Не пишите без крайней нужды один тип дважды.